Doc/howto/enum.rst

```diff
@@ -1050,10 +1050,14 @@ If you give your enum subclass extra methods, like the `Planet`_
 class below, those methods will show up in a :func:`dir` of the member,
 but not of the class::
 
-    >>> dir(Planet)                         # doctest: +SKIP
-    ['EARTH', 'JUPITER', 'MARS', 'MERCURY', 'NEPTUNE', 'SATURN', 'URANUS', 'VENUS', '__class__', '__doc__', '__members__', '__module__']
-    >>> dir(Planet.EARTH)                   # doctest: +SKIP
-    ['__class__', '__doc__', '__module__', 'mass', 'name', 'radius', 'surface_gravity', 'value']
+    >>> dir(Planet)
+    ['EARTH', 'JUPITER', 'MARS', 'MERCURY', 'NEPTUNE', 'SATURN', 'URANUS', 'VENUS', '__class__', '__contains__', '__doc__', '__getitem__', '__init_subclass__', '__iter__', '__len__', '__members__', '__module__', '__name__', '__qualname__']
+    >>> dir(Planet.EARTH)
+    ['EARTH', 'JUPITER', 'MARS', 'MERCURY', 'NEPTUNE', 'SATURN', 'URANUS', 'VENUS', '__class__', '__doc__', '__eq__', '__hash__', '__module__', 'mass', 'name', 'radius', 'surface_gravity', 'value']
+
+.. versionchanged:: 3.11
+    Additional ``__dunder__`` names are returned for enums and enum members.
+    Calling :func:`dir` on an enum member also returns the other members.
```

Comment:
We don't need versionchanged in a How To document 

```suggestion
```

Reply:
Should I remove it? Note that there are 13 other occurrences of versionadded/versionchanged in the document